pkg/hubble/relay/server.go

```diff
@@ -60,11 +53,32 @@ func NewServer(options ...relayoption.Option) (*Server, error) {
 	}
 	logger := logging.DefaultLogger.WithField(logfields.LogSubsys, "hubble-relay")
 	logging.ConfigureLogLevel(opts.Debug)
+
+	pool, err := pool.NewManager(
```

Comment:
nit: I find a bit confusing to declare a local var `pool` when `github.com/cilium/cilium/pkg/hubble/relay/pool` is imported.

Reply:
I renamed it to `pm`.